### PR TITLE
fragment orders

### PR DESCRIPTION
**Pull request by lamvuvan**

### CODE REVIEW

File: app/src/main/res/layout/item_order.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="7"
+            android:orientation="vertical"
+            android:weightSum="2">
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="0dp"
+                android:layout_weight="1"/>
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_order.xml]: WARNING: [NestedWeights] Nested weights are bad for performance
 Layout weights require a widget to be measured twice. When a LinearLayout with non-zero weights is nested inside another LinearLayout with non-zero weights, then the number of measurements increase exponentially.

File: app/src/main/java/com/framgia/awesomecity/screen/orders/OrderViewModel.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    public Order getOrder(){
+        return mOrder;
+    }
+
+    @Bindable
+    public String getWaitTime(){
+        Calendar calendar = Calendar.getInstance();
+        int minutes = calendar.get(Calendar.MINUTE);
+        return minutes + " minutes";
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/screen/orders/OrderViewModel.java]: hardcode

File: app/src/main/java/com/framgia/awesomecity/screen/orders/OrdersFragment.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import com.framgia.awesomecity.R;
+import com.framgia.awesomecity.databinding.FragmentOrdersBinding;
+
+/**
+ * Created by lamvu on 8/8/2017.
+ */
+
+public class OrdersFragment extends Fragment {
+
+    private static final String FRAGMENT_NAME = "FRAGMENT_NAME";
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/screen/orders/OrdersFragment.java]: 2.10 String constants, naming and values
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md
e xem llai cách đặt tên Field Name Prefix
--



Field Name Prefix
--




Field Name Prefix
--

File: app/src/main/java/com/framgia/awesomecity/screen/orders/OrdersPresenter.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    public void onStop() {
+    }
+
+    @Override
+    public void getOrders() {
+        fetchOrders();
+    }
+
+    private void fetchOrders(){
+        mCompositeDisposable.add(mOrdersRepository.getOrders()
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/screen/orders/OrdersPresenter.java]: e chưa quản lý Observeble bằng mCompositeDisposable

File: app/src/main/java/com/framgia/awesomecity/screen/orders/OrdersPresenter.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribeWith(new DisposableObserver<List<Order>>() {
+                    @Override
+                    public void onNext(@NonNull List<Order> orders) {
+                        mViewModel.onGetOrdersSuccess(orders);
+                    }
+
+                    @Override
+                    public void onError(@NonNull Throwable e) {
+
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/screen/orders/OrdersPresenter.java]: error cần thông báo tới user

File: app/src/main/res/layout/item_order.xml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        android:layout_height="@dimen/dp_100"
+        android:weightSum="10">
+        <TextView
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="3"/>
+        <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="match_parent"
+            android:layout_weight="7"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_order.xml]: Nested attr bad performance

Comment (lamvuvan) [inline app/src/main/res/layout/item_order.xml]: warning em đã sửa từ lần trước rồi ạ

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -4,4 +4,5 @@
     <string name="text_username_hint">Enter an username</string>
     <string name="text_password_hint">Password</string>
     <string name="text_refresh">Refresh</string>
+    <string name="error_fetch_data">No data available.</string>
```
Comment (DoanVanToan) [inline app/src/main/res/values/strings.xml]: WARNING: [UnusedResources] The resource `R.string.error_fetch_data` appears to be unused
 Unused resources make applications larger and slow down builds.

**Pull request merged**